yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -35,12 +35,14 @@ Arguments:
   assessment_metadata_dir     The directory path where the assessment metadata will be stored.
                               This script will attempt to create the directory if it does not exist.
 
-  iops_capture_interval   This argument is used to configure the interval for measuring the IOPS 
-                               metadata on source (in seconds). (Default 120)
-                            
+  pgss_enabled                This argument is used to determine whether pg_stat_statements extension
```

Comment:
nit: "This argument is used to" is redundant. 
Extract and analyze statements from pg_stat_statements table